src/Core/Addon/Module/ModuleManager.php

```diff
@@ -317,6 +307,34 @@ public function install($source)
         return $result;
     }
 
+
+    /**
+     * Execute post install
+     *
+     * @param string $moduleName
+     *
+     * @return bool true for success
+     */
+    public function postInstall(string $moduleName): bool
```

Comment:
Could you test it ?

Reply:
Same I can't test it, too much dependency from the database and fs.
The current state of these files are not designed for unit testing :sob: